workshops/views.py

```diff
@@ -136,6 +140,58 @@ def person_details(request, person_id):
     return render(request, 'workshops/person.html', context)
 
 
+def person_bulk_add(request):
+    if request.method == 'POST':
+        form = PersonBulkAddForm(request.POST, request.FILES)
+        if form.is_valid():
+            try:
+                persons_tasks = _upload_person_task_csv(request, request.FILES['file'])
+            except csv.Error as e:
+                messages.add_message(request, messages.ERROR, "Error processing uploaded .CSV file: {}".format(e))
+            else:
+                for entry in persons_tasks:
```

Comment:
There should definitely be a transaction here, and adding People and 
Task records to the DB should only happen after they've been validated 
and the user has pressed 'yes' on the confirmation form.